src/lib/isMobilePhone.js

```diff
@@ -105,6 +105,7 @@ const phones = {
   'hu-HU': /^(\+?36|06)(20|30|31|50|70)\d{7}$/,
   'id-ID': /^(\+?62|0)8(1[123456789]|2[1238]|3[1238]|5[12356789]|7[78]|9[56789]|8[123456789])([\s?|\d]{5,11})$/,
   'ir-IR': /^(\+98|0)?9\d{9}$/,
+  'is-IS': /^(\+354)(61[1-8]|62[0-6]|63[02]|64[4679]|65[0159]|66[0-69]|670|68[06-8]|69[0-9]|7[6-8][0-9]|79[0-3]|82[0-59]|8[3-6][0-9]|88[238]|89[0-9])\d{4}$/,
```

Comment:
I don't know if we said anything about simplifying this regex?

Reply:
There's a little bit that we could simplify without changing the results, but I don't think that's worth it for now. It's like @pano9000 mentioned https://github.com/validatorjs/validator.js/pull/2139#issuecomment-1369105904 "there are quite a few exceptions in between those ranges"